### ISSUE TITLE
Any way to trigger mergebot for non-github repos?

### ISSUE THREAD

**Issue by cgeoga** [JuliaRegistries/General#66483]
I use `LocalRegistry.jl` to generate PRs to merge packages that aren't hosted on github to the general registry (based on advice I got in [#49803](https://github.com/JuliaRegistries/General/issues/49803)). It works like a charm, but every time I merge something I have to bug somebody to click merge. But that can be a bit slow---for example, I'm trying to [register another package](https://github.com/JuliaRegistries/General/pull/65487) that has all tests passing but has been sitting as an open PR for half a month now.

I don't like hounding people and I'm sure the people I'm hounding don't like it either. Considering that these PRs are still auto-generated just like they would be from JuliaHub, is there any way to make it possible for people without merge rights in this repo to trigger merge bot if all the tests for it pass? Or, like, would it be possible to run the merge bot tests, and then if they all pass put a "just click merge" tag on it or something so that it is not so easy to miss in the sea of auto-generated PRs?

I recognize that the simplest way to avoid hounding people is just to host the code on github. That would be sort of disappointing, though, because it really seems like all the machinery for these PRs to get checked automatically is there. I also have tried using JuliaHub to create the PRs for packages, but for code that isn't on github I don't see how to escape the error that JuliaHub can't prove that I own the repo. 

If triggering merge bot for auto-generated new package PRs is not appealing, which I would understand, would it at least be possible for things that are already registered? I already want to try and merge a new point release for the package I have that is already in the general registries, but I've been waiting to make that PR so that I'm not bugging individuals on too many PRs at once. 

I recognize that you guys are juggling a lot of problems in maintaining this registry, and I am very appreciative of all the work that has gone into making it and `Pkg` such a pleasure to use. So if the answer here is that you really don't want to make any of this possible, that would be disappointing but very understandable. In that case, I suppose I'd just close my open PR and move it to a github mirror so I can use JuliaHub. But I would appreciate any input on how to handle point releases in the package that I already have registered here without having to bug the kind people who were helpful to me in the past, which is sort of what I have defaulted to doing. I suppose I could also move that to a github mirror and then open a manual PR that changes the URL, but that seems like a more manual surgery that maybe is not advisable.

**Comment by DilumAluthge** [comment]
This is a great question. I'm swamped right now, but I'll write up my thoughts when I get a chance - ping me if I don't follow up within a few weeks?

**Comment by DilumAluthge** [comment]
BTW, to get you unblocked in the short-term, you can ask for manual merges in the `#pkg-registration` channel on the [Julia Slack](https://julialang.org/slack/). Sometimes you can get a faster response that way.

**Comment by cgeoga** [comment]
That sounds great, thanks! I'm curious to hear your thoughts and will circle back in a few weeks. Thanks also for the slack link---I'll give it a shot!

**Comment by DilumAluthge** [comment]
I think this is a duplicate of https://github.com/JuliaRegistries/General/issues/116716.

If it's not the same issue, let me know, and I'll reopen this one.

**Comment by DilumAluthge** [comment]
Duplicate of #116716
